src/components/pages/PassportChangesRedirect/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { routes } from 'src/constants/routes';
+import { Redirect } from 'react-router';
+import { createRouteUrl, IParsedQueryString } from 'src/utils/nav';
+import { parse } from 'query-string';
+
+export const PassportChangesRedirect = ({ location }) => {
+  const newLocation = { ...location };
+  const { pathname } = newLocation;
+  const regex = new RegExp(routes.LegacyPassportChanges);
+  newLocation.pathname = pathname.replace(regex, routes.Passport);
+  const parsedQueryString = parse(newLocation.search);
+
+  const routeUrl = createRouteUrl(newLocation, routes.Passport, parsedQueryString as IParsedQueryString);
+
+  return <Redirect from={routes.LegacyPassportChanges} to={routeUrl} />;
```

Comment:
I think we are losing passport address here.

Let's say user comes with url:  
`/passport-changes/0x123456789?network=ropsten`

But in `<Redirect`, we see that it redirects to `routeUrl`, which is constructed like:
`${routes.Passport}${queryString}`, but we lose passport address

Reply:
Yeah, need to think how to proper use the existing method :)